pkg/config/load.go

```diff
@@ -33,6 +33,21 @@ type Prowgen struct {
 	Expose bool `json:"expose,omitempty"`
 	// Rehearsals declares any disabled rehearsals for jobs
 	Rehearsals Rehearsals `json:"rehearsals,omitempty"`
+	// Set which cluster and architecture should the images be promoted from
+	Images Images `json:"postsubmit_images"`
+}
+
+type Images struct {
+	// amd64 is the default arch, falsify this field to skip it
+	SkipDefaultArch bool   `json:"skip_default_arch,omitempty"`
+	Architectures   []Arch `json:"architectures"`
+}
+type Arch struct {
+	// The name of the architecture, it will be used as suffix for the job
+	// Ex.: openshift-ci-tools-master-images-${arch.Name} -> openshift-ci-tools-master-images-arm64
+	Name string `json:"name,omitempty"`
+	// The cluster that will run the promotion
+	Cluster string `json:"cluster,omitempty"`
```

Comment:
This is not needed. We should hardcode the clusters/architectures and validate the config as well.

Reply:
By hardcoding clusters/architectures, do you mean something like this one below?
```golang
func determineCluster(arch string) (string, error) {
  switch arch {
  case "arm64":
    return "arm01", nil
  case "powervs":
    return "pw01", nil
  // More cases here
  // ...
  default:
    return "", fmt.Errorf("arch %s is not supported", arch)
  }
}
```

Reply:
Yea, but let's have a constant for each arch and cluster. I think we already do somewhere. Techically, we should have a way to connect the arch with clusters. Now we have `arm64` --> `arm01` cluster, but we are planning to add more.